woocommerce/woocommerce
# Corrected path for publishing commands

### All Submissions:


### Changes proposed in this Pull Request:

- typo in the path in previous commit

### How to test the changes in this Pull Request:

1. You can run `pnpm run publish-packages:prod --filter @woocommerce/admin-library` if you want to confirm that the path is correct, but might want to cancel the publishing unless you intend to

### FOR PR REVIEWER ONLY:
